packages/bson-transpilers/symbols/java/templates.yaml

```diff
@@ -75,11 +86,14 @@ Templates:
                     case 'collation':
                         warning = '\n *\n * Warning: translating collation to Java not yet supported, so will be ignored';
                         return s;
+                    case 'exportMode':
+                        return s;
                     default:
+                        if (!spec[k]) return s;
                         return `${s}
             .${k}(${k})`;
                 }
-            }, 'FindIterable<Document> result = collection.find(filter)');
+            }, `FindIterable<Document> result = collection.${driverMethod}(filter)`);
```

Comment:
[Looking at the java docs](https://www.mongodb.com/docs/drivers/java/sync/current/usage-examples/updateMany/), the result here should also be updated for the method, instead of `FindIterable`, it seems to be `UpdateResult` / `DeleteResult` depending on the method

Reply:
You are right, taking a look now. Some languages you use `var` but for Java and others I need to take a look.